LSP-vue.sublime-settings

```diff
@@ -2,20 +2,24 @@
 	"languages": [
 		{
 			"languageId": "vue",
+			// ST3
 			"scopes": ["text.html.vue"],
 			"syntaxes": ["Packages/Vue Syntax Highlight/Vue Component.sublime-syntax"],
+			// ST4
+			"document_selector": "text.html.vue",
```

Comment:
Keeping compatibility with ST3 while also having the schema mark ST3 properties as deprecated results in there being warnings in the default configuration file. Not ideal really. Separating the package into ST3 and ST4 is not great either as it requires more maintenance.